src/misc/sphere-collider.js

```diff
@@ -68,10 +69,11 @@ module.exports = {
 
       // AABB collision detection
       function intersect (el) {
-        var radius,
-            mesh = el.getObject3D('mesh');
+        var radius, mesh;
+        if ( !el.getObject3D ) { return; }
```

Comment:
Could you replace this check with `if (!el.isEntity) { return; }` ?

(The result should be the same)